earth_enterprise/tutorial/FusionTutorial/download_tutorial.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+TMPFILE=/tmp/FusionTutorial-Full.tar.gz
+SCRIPTDIR=`dirname $0`
```

Comment:
@tst-ppenev I'm sorry, Pavel, but I'm not sure what you mean by quoting here - please illustrate so I get it right.